crates/bevy_core_pipeline/src/smaa/mod.rs

```diff
@@ -95,17 +95,21 @@ const SMAA_SEARCH_LUT_TEXTURE_HANDLE: Handle<Image> = Handle::weak_from_u128(318
 /// Adds support for subpixel morphological antialiasing, or SMAA.
 pub struct SmaaPlugin;
 
-/// Add this component to a [`bevy_render::camera::Camera`] to enable subpixel
-/// morphological antialiasing (SMAA).
+/// A component for enabling Subpixel Morphological Anti-Aliasing (SMAA)
+/// for a [`bevy_render::camera::Camera`].
 #[derive(Clone, Copy, Default, Component, Reflect, ExtractComponent)]
 #[reflect(Component, Default)]
-pub struct SmaaSettings {
+#[doc(alias = "SubpixelMorphologicalAntialiasing")]
```

Comment:
Whoops, missed capitalization here
```suggestion
#[doc(alias = "SubpixelMorphologicalAntiAliasing")]
```